decayAmplitude/diffractiveDissVertex.cc

```diff
@@ -336,3 +338,10 @@ diffractiveDissVertex::printPointers(ostream& out) const
 	    << "recoil particle = " << recoil()    << endl;
 	return out;
 }
+
+double diffractiveDissVertex::getTprime() const{
+	const TLorentzVector lvBeam = TLorentzVector(beam()->lzVec());
```

Comment:
The t' function boosts the 4-vectors, but the class method should be const, thus the vectors have to be copied. References will be added